src/ProjectTemplates/Web.ProjectTemplates/content/Grpc-Application/Grpc-Application.Client/Program.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Greet;
+using Grpc.Core;
+
+namespace Company.ClientApplication
+{
+    public class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var channel = new Channel("localhost:50051", ChannelCredentials.Insecure);
```

Comment:
Should the port be hardcoded here?

Reply:
No

Reply:
How do we plan on configuring the client then? It's a console app so we'd have to add all the extensions.configuration code?